lotemplate/classes.py

```diff
@@ -77,6 +78,31 @@ def restart(self) -> None:
         self.__init__(self.host, self.port)
 
 
+class IfStatement:
+    """
+    Class representing an if statement in a template libreoffice
+    """
+    start_regex = r'\[\s*if\s*\$(\w+(\(.+?\))?)\s*(\=\=|\!\=)\s*([\w\-\.\,]+)\s*\]'
+    end_regex = r'(\[\s*endif\s*\])$'
+
+    def __init__(self, if_string):
+        self.if_string = if_string
+        match = re.search(self.__class__.start_regex, if_string, re.IGNORECASE)
```

Comment:
```py
match = re.search(self.start_regex, if_string, re.I)
```

Reply:
I keep the re.IGNORECASE to keep the code more explicit.

I removed the __class__ keyword that seemed to be useless